src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -75,6 +79,7 @@ public ConversionUnit SelectedFromConversion
             }
         }
 
+        [JsonProperty("ToConversion")]
```

Comment:
Are these two properties used separately from MetricConversion?  That sounds like it could be very confusing, and problematic.